libs/net/kafka/Interfaces/IKafkaListener`.cs

```diff
@@ -44,11 +44,6 @@ public interface IKafkaListener<TKey, TValue>
     /// get - An array of topics this consumer is subscribed to.
     /// </summary>
     public string[] Topics { get; }
-
-    /// <summary>
-    /// get/set - The max threads for running service.
-    /// </summary>
-    public int MaxThreads { get; set; }
```

Comment:
@CelloLiu This did not work.  The services that attempted to change the value were ignored.